FinSearchUnified/Resources/config.xml

```diff
@@ -83,6 +83,13 @@
                 <readOnly>true</readOnly>
             </options>
         </element>
+        <element type="button" scope="shop">
```

Comment:
Please check how we can make this work for Shopware < 5.2.17 If you can not figure it out or it just does not work, you should still be able to change/install the Plugin when using Shopware < 5.2.17, therefore the button may be removed completely in that case.